docs/tutorial/force_calibration.rst

```diff
@@ -104,3 +118,89 @@ We can plot the calibration by calling::
     calibration.plot()
 
 .. image:: force_calibration_fit.png
+
+Hydrodynamics
+-------------
+
+While the idealized Lorentzian model discussed in the previous section provides a reasonable first description of the physical power spectrum, better models for this spectrum exist.
+
+The idealized model is based on the assumption that the drag force is only proportional to the bead's velocity.
+This assumption is realistic when the bead moves at a constant speed with respect to the fluid.
+When oscillating a bead however, this frictional force becomes frequency dependent.
+
+Equations have been derived by other authors which provide frequency dependent drag coefficients :cite:`tolic2006calibration`.
+These equations can be used to fit a hydrodynamically correct power spectrum:
+
+.. math::
+
+    P_{hydro}(f) = \frac{D Re(\gamma / \gamma_0)}{\pi^2 \left(\left(f_{c,0} + f Im(\gamma/\gamma_0) - f^2/f_{m, 0}\right)^2 + \left(f Re(\gamma / \gamma_0)\right)^2\right)}
+
+where
+
+.. math::
+
+    f_{c, 0} = \frac{\kappa}{2 \pi \gamma_0}
+
+and
+
+.. math::
+
+    f_{m, 0} = \frac{\gamma_0}{2 \pi m}
+
+with :math:`m` the mass of the bead and :math:`\gamma` the frequency dependent drag coefficient.
+Assuming a measurement deep in bulk, these are given by:
+
+.. math::
+
+    \gamma_{stokes} = \gamma_0 \left(1 + (1 - i)\sqrt{\frac{f}{f_{\nu}}} - \frac{2}{9}\frac{f}{f_{\nu}} i\right)
+
+Where :math:`f_{\nu}` is given by :math:`4 \nu/(\pi d^2)` with :math:`\nu` the kinematic viscosity.
```

Comment:
What is the definition in words for f_{\nu}? This frequency determines when a hydrodynamic correction becomes important. Could you explain in a few words why that is? Do you get turbulence when you go above this frequency?

Reply:
Ah yes, I am familiar with the term penetration depth, I think you can leave out the definition. The way it is written now, it looks like the formula at the end of the sentence ( 4 \nu/(\pi d^2) is the penetration depth. 

How about: 

Where :math:`f_{\nu}` is the frequency at which the penetration depth in the liquid of the bead’s linear harmonic motion equals the radius of the bead, 
`f_{\nu}=4 \nu/(\pi d^2)`
with \nu the kinematic viscosity

Reply:
Sounds good to me! :)